### ISSUE TITLE
How can i use Microsoft login 

### ISSUE THREAD

**Issue by Pratiklocho** [EddyVerbruggen/nativescript-plugin-firebase#1273]
i didn't find anything about Microsoft login in doc can anyone can help me how can i achieve  login with microsoft?

**Comment by EddyVerbruggen** [comment]
Can you point me at the native SDK docs for microsoft login?

**Comment by Pratiklocho** [comment]
https://docs.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-android

**Comment by EddyVerbruggen** [comment]
What does that have to do with this plugin?

**Comment by Pratiklocho** [comment]
how i can implement Microsoft login in using your plugin ?

**Comment by EddyVerbruggen** [comment]
This plugin exposes the ios and android Firebase sdks. What feature of those sdks are you referring to? Please provide at least a link to the Firebase docs.

**Comment by Pratiklocho** [comment]
I want this 

https://firebase.google.com/docs/auth/android/microsoft-oauth
https://firebase.google.com/docs/auth/ios/microsoft-oauth

**Comment by EddyVerbruggen** [comment]
Ah nice, that must be a recent addition.

**Comment by Pratiklocho** [comment]
yes it is. so now when i can expect this thing add in your package😜  ?

**Comment by EddyVerbruggen** [comment]
Depends on when someone thinks they need this feature and implement it. And that someone may be me, but not right now.

**Comment by CaptainDingle** [comment]
+1 for the Microsoft login. We are using Azure AD and have this working with Firebase for web app. It would be awesome if it made it to this plugin at some point for mobile. Thanks!

**Comment by mhtsharma948** [comment]
Adding this feature would be a +1. My web app uses Azure AD and need to use that same in Mobile application. Can you provide an ETA for this feature?

**Comment by LovepreetSinghMe** [comment]
Expecting login from microsoft in this plugin. Can you please tell me ETA for the feature.

**Comment by EddyVerbruggen** [comment]
@LovepreetSinghMe I just released "Sign in with Apple" yesterday because that's a mandatory-ish requirement for Apple, but I have not started on Microsoft login yet. My PR's are wide open though.

**Comment by Ianc22** [comment]
Hey @EddyVerbruggen any update to this? We also now need MS login as we're integrating MS Calendar in app. 

I'm sure you're aware, but here's the list of Firebase sign-in providers
![image](https://user-images.githubusercontent.com/4292433/72473156-0129b780-37de-11ea-9b47-4f86ce87ffcb.png)

**Comment by AmithRamesh** [comment]
Hi @EddyVerbruggen, a friendly remainder and request to add Microsoft authentication.
If this is not on priority, can you/ anyone please suggest any alternative to add Microsoft authentication to Nativescript-Angular app with firebase.

**Comment by shuchow** [comment]
+1 on this.  Would help us a lot.

**Comment by ReneMueller4P** [comment]
HI @EddyVerbruggen @NathanWalker,
is it in planning to implement microsoft as authentication provider? Or are there other alternatives ?
